src/app/georeferencer/qgsgeorefmainwindow.cpp

```diff
@@ -401,16 +401,22 @@ void QgsGeoreferencerMainWindow::generateGDALScript()
 void QgsGeoreferencerMainWindow::setAddPointTool()
 {
   mCanvas->setMapTool( mToolAddPoint );
+  QgisApp::instance()->mapCanvas()->setMapTool( mPrevQgisMapTool );
```

Comment:
I think we should check here that mPrevQgisMapTool is not nullptr. Otherwise I believe that on creation QgsGeoreferencerMainWindow will call setAddPointTool with mPrevQgisMapTool as null, and cause the qgis window to lose its current map tool immediately